go/libraries/doltcore/servercfg/yaml_config.go

```diff
@@ -276,6 +342,229 @@ func (cfg YAMLConfig) String() string {
+	if withPlaceholders.BehaviorConfig.DisableClientMultiStatements == nil {
+		withPlaceholders.BehaviorConfig.DisableClientMultiStatements = ptr(false)
+	}
+	if withPlaceholders.BehaviorConfig.EventSchedulerStatus == nil {
+		withPlaceholders.BehaviorConfig.EventSchedulerStatus = ptr("OFF")
+	}
+
+	if withPlaceholders.ListenerConfig.TLSKey == nil {
+		withPlaceholders.ListenerConfig.TLSKey = ptr("key.pem")
+	}
+	if withPlaceholders.ListenerConfig.TLSCert == nil {
+		withPlaceholders.ListenerConfig.TLSCert = ptr("cert.pem")
+	}
+	if withPlaceholders.ListenerConfig.RequireSecureTransport == nil {
+		withPlaceholders.ListenerConfig.RequireSecureTransport = ptr(false)
+	}
+	if withPlaceholders.ListenerConfig.Socket == nil {
+		withPlaceholders.ListenerConfig.Socket = ptr(DefaultUnixSocketFilePath)
+	}
+
+	if withPlaceholders.MetricsConfig.Labels == nil {
+		withPlaceholders.MetricsConfig.Labels = map[string]string{
+			"label1": "value1",
+			"label2": "2",
+			"label3": "true",
+		}
+	}
+	if withPlaceholders.MetricsConfig.Host == nil {
+		withPlaceholders.MetricsConfig.Host = ptr("123.45.67.89")
+	}
+	if withPlaceholders.MetricsConfig.Port == nil {
+		withPlaceholders.MetricsConfig.Port = ptr(9091)
+	}
+
+	if withPlaceholders.RemotesapiConfig.Port_ == nil {
+		withPlaceholders.RemotesapiConfig.Port_ = ptr(8000)
+	}
+	if withPlaceholders.RemotesapiConfig.ReadOnly_ == nil {
+		withPlaceholders.RemotesapiConfig.ReadOnly_ = ptr(false)
+	}
+
+	if withPlaceholders.ClusterCfg == nil {
+		withPlaceholders.ClusterCfg = &ClusterYAMLConfig{
+			StandbyRemotes_: []StandbyRemoteYAMLConfig{
+				StandbyRemoteYAMLConfig{
+					Name_:              "standby_replica_one",
+					RemoteURLTemplate_: "https://standby_replica_one.svc.cluster.local:50051/{database}",
+				},
+				StandbyRemoteYAMLConfig{
+					Name_:              "standby_replica_two",
+					RemoteURLTemplate_: "https://standby_replica_two.svc.cluster.local:50051/{database}",
+				},
+			},
+			BootstrapRole_:  "primary",
+			BootstrapEpoch_: 1,
+			RemotesAPI: ClusterRemotesAPIYAMLConfig{
+				Addr_:    "127.0.0.1",
+				Port_:    50051,
+				TLSKey_:  "remotesapi_key.pem",
+				TLSCert_: "remotesapi_chain.pem",
+				TLSCA_:   "standby_cas.pem",
+				URLMatches: []string{
+					"https://standby_replica_one.svc.cluster.local",
+					"https://standby_replica_two.svc.cluster.local",
+				},
+				DNSMatches: []string{
+					"standby_replica_one.svc.cluster.local",
+					"standby_replica_two.svc.cluster.local",
+				},
+			},
+		}
+	}
+
+	if withPlaceholders.Vars == nil {
+		withPlaceholders.Vars = []UserSessionVars{
+			UserSessionVars{
+				Name: "root",
+				Vars: map[string]interface{}{
+					"dolt_show_system_tables": 1,
+					"dolt_log_level":          "warn",
+				},
+			},
+		}
+	}
+
+	if withPlaceholders.SystemVars_ == nil {
+		withPlaceholders.SystemVars_ = map[string]interface{}{
+			"dolt_transaction_commit": 1,
+			"dolt_log_level":          "info",
+		}
+	}
+
+	if withPlaceholders.Jwks == nil {
```

Comment:
Default for this field should be empty (it's advanced and poorly documented, currently only used for our hosting service)